src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
 
 WARNING: Once the upgrade is performed and the new version started for the first time, the ProActive server will upgrade <<_glossary_catalog,Catalog>> objects to the new version. This procedure ensures that any object modified manually by a user will not be upgraded.
 
-=== Install and Start ProActive using Docker Containers
+=== Automated ProActive Installation with Docker Compose
 ProActive could be easily deployed and started within a fully containerized environment. For this purpose, we provide a ready-to-use Docker Compose configuration that enables you running ProActive Workflows and Scheduling as a containerized service.
```

Comment:
ProActive Workflows and Scheduling --> ProActive Workflows & Scheduling (PWS) 
because you've used this acronym to name certain properties

Reply:
ok